frontend/src/translations/vi/common.json

```diff
@@ -556,7 +552,14 @@
     },
     "ListHandler": {
       "last_block": "Chiều cao khối mới nhất là "
-    }
+    },
```

Comment:
We don't have this data in vietnamese.
@frol Can we get it before merging this PR?

Reply:
What data do you refer to? Do you mean that some translations are lacking? I suggest we don't do anything right now since we will likely need more translation refactoring after we do UX revamp

Reply:
I refer to a couple of keys I added to the json here. Ok, I'll keep text in english then.